content/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "CytoData Society"
+---
+
+![CytoData banner](/./main_files/cytodata-banner.png)
+The CytoData society (CytoDS) builds and maintains an active community around image-based profiling of biological phenotypes induced by genetic, chemical or other perturbations of biological systems.
+
+CytoData Society is an unregistered society meaning that the society does not have an official status in any country. Instead we are an active community of researchers and developers. We consider everyone in our mailing list as a member of the society.
```

Comment:
Move to bottom of page and give it a \<div style="background:#808080">\</div>

Reply:
Do you mean below the subscriptions button?

Reply:
yes

Reply:
I did test some thing with a darker background but nothing was nice. Italic seems to have the cleanest but still clear effect

Reply:
also good